resources/user-agents/browsers/mozilla-browser/mozilla-browser-0-6-1-8.json

```diff
@@ -0,0 +1,85 @@
+{
+  "division": "Firefox #MAJORVER#.#MINORVER#",
+  "versions": ["1.8", "1.7", "1.6", "1.5", "1.4", "1.3", "1.2", "1.1", "1.0", "0.9", "0.8", "0.7", "0.6"],
+  "sortIndex": 3172,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Mozilla #MAJORVER#.#MINORVER#",
+      "engine": "Gecko_Dynamic",
+      "device": "general Desktop",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Mozilla #MAJORVER#.#MINORVER#",
+        "Browser": "Mozilla",
+        "Browser_Maker": "Mozilla Foundation",
+        "Browser_Type": "Browser",
+        "Version": "#MAJORVER#.#MINORVER#",
+        "MajorVer": "#MAJORVER#",
+        "MinorVer": "#MINORVER#",
+        "Frames": "true",
+        "IFrames": "true",
+        "Tables": "true",
+        "Cookies": "true",
+        "JavaScript": "true",
+        "CssVersion": "2"
+      },
+      "children": [
+        {
+          "match": "Mozilla/5.0 (#PLATFORM# rv:#MAJORVER#.#MINORVER#*) Gecko/200?????",
+          "device": "Linux Desktop",
+          "platforms": [
+            "Linux_64", "Linux_32", "Linux_PPC", "Linux"
+          ]
+        },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM# rv:#MAJORVER#.#MINORVER#*) Gecko/200????? Debian/#MAJORVER#.#MINORVER#*",
```

Comment:
That said, I did add these specific Linux variants to catch some Mozilla builds on these linux distributions (mainly because there was a handful of these agents in the issue's files). I was careful to add the matching versions here, as those agents did exhibit this behavior. Adding just an asterisk here could cause the pattern to match agents it shouldn't.